src/features/views/l10n/messageIds.ts

```diff
@@ -350,6 +350,7 @@ export default makeMessages('feat.views', {
       warning2: m<{ shareLink: ReactElement }>(
         'You can {shareLink} within Zetkin. Exporting makes sense when you want to copy data to another system.'
       ),
+      local_bool: m('Yes'),
```

Comment:
The name `local_bool` implies that this only is relevant for the `local_bool` column type (which is just one of several column types that produce boolean values).